services_app/src/main/AndroidManifest.xml

```diff
@@ -112,7 +112,7 @@
         <activity
             android:name=".MainActivity"
             android:label="@string/app_name"
-            android:theme="@style/AppTheme">
+            android:theme="@style/Opendatakit">
```

Comment:
it would probably be better to rename this style to ODKX (and yes that will require a change to androidlibrary)